CHANGELOG.md

```diff
@@ -1,19 +1,32 @@
 Changelog
 =========
 
-## [Unreleased]
+## [v1.1.0]
+
+### New feature support
+* BGP (bgp, bgp_af, bgp_neighbor, bgp_neighbor_af)
+  * Added support for bgp (@mikewiebe)
+  * Added support for bgp_af (@richwellum)
+  * Added support for bgp_neighbor (@jyang09)
+  * Added support for bgp_neighbor_af (@chrisvanheuveln)
```

Comment:
"Added support for " seems redundant. How about this instead?

~~~
* BGP
  * bgp (@mikewiebe)
  * bgp_af (@richwellum)
  * bgp_neighbor (@jyang09)
  * bgp_neighbor_af (@chrisvanheuveln)
~~~